core/src/main/java/com/linecorp/armeria/internal/server/DefaultServiceRequestContext.java

```diff
@@ -116,6 +116,7 @@
 
     @Nullable
     private String strVal;
+    private boolean shouldReportUnhandledExceptions = true;
```

Comment:
How about moving this field next to `log`? The order of member fields should roughly match the order of methods for readability.